### PR TITLE
updating reporting and works json with updates from sm-4295 and sm-4373

### PR DESCRIPTION
**Pull request by shaunganley**

### CODE REVIEW

File: docs/api-documentation/V1.12/json/export-swagger.json [hunk 1]
```diff
@@ -3,7 +3,227 @@
 	"consumes": [
 		"application/json"
 	],
-	"definitions": {},
+	"definitions": {
```
Comment (KylerD) [inline docs/api-documentation/V1.12/json/export-swagger.json]: @peadarkelly best you review this part specifically

File: docs/api-documentation/V1.12/json/export-swagger.json [hunk 2]
```diff
@@ -3,7 +3,227 @@
...
+				"acknowledged",
+				"accepted",
+				"accepted_fixed",
+				"rejected",
+				"resolved_by_ha",
+				"cancelled"
+			],
+			"type": "string"
+		},
+		"Section81CSVExportRequest": {
```
Comment (peadarkelly) [inline docs/api-documentation/V1.12/json/export-swagger.json]: Severity needs added to this

File: docs/api-documentation/V1.12/json/export-swagger.json (hunk 1 above)
Comment (peadarkelly) [reply docs/api-documentation/V1.12/json/export-swagger.json]: Missing `permits/csv`

File: docs/api-documentation/V1.12/json/reporting-swagger.json [hunk 3]
```diff
@@ -566,50 +714,59 @@
...
 				}
 			},
 			"required": [
+				"site_number",
 				"work_reference_number",
-				"forward_plan_reference_number",
 				"promoter_organisation",
 				"highway_authority",
+				"street_name",
+				"area_name",
```
Comment (peadarkelly) [inline docs/api-documentation/V1.12/json/reporting-swagger.json]: I don't think area should be required as it's set to nullable in the database. SM-3383 also states that it's optional.

File: docs/api-documentation/V1.12/json/reporting-swagger.json [hunk 4]
```diff
@@ -901,29 +994,32 @@
...
 			"type": "object"
 		},
-		"ReinstatementType": {
+		"Section81Severity": {
 			"enum": [
-				"excavation",
-				"bar_holes",
-				"core_holes",
-				"pole_testing"
+				"HIGH",
```
Comment (peadarkelly) [inline docs/api-documentation/V1.12/json/reporting-swagger.json]: these will be lower case

File: docs/api-documentation/V1.12/json/work-swagger.json [hunk 5]
```diff
@@ -4356,6 +4357,44 @@
 			],
 			"type": "object"
 		},
+		"Section81WorkType": {
+			"enum": [
+				"PERMIT_WORKS",
```
Comment (peadarkelly) [inline docs/api-documentation/V1.12/json/work-swagger.json]: will be lowercase as per @matthewdoreilly 's change

File: docs/api-documentation/V1.12/json/export-swagger.json [hunk 6]
```diff
@@ -28,14 +248,267 @@
...
+				],
+				"responses": {
+					"200": {
+						"description": "Ok",
+						"schema": {
+							"$ref": "#/definitions/CSVExportResponse"
+						}
+					}
+				},
+				"description": "See API specification Resource Guide > Reporting API > CSV for more information\nDeprecated - this will be removed in a future release",
```
Comment (peadarkelly) [inline docs/api-documentation/V1.12/json/export-swagger.json]: Reporting API needs replaced with Data Export API throughout this file

File: docs/api-documentation/V1.12/json/work-swagger.json [hunk 7]
```diff
@@ -1544,16 +1591,11 @@
 		},
 		"WorkResponse": {
```
Comment (peadarkelly) [inline docs/api-documentation/V1.12/json/work-swagger.json]: historical_permit_reference is missing

**Pull request merged**